.github/ISSUE_TEMPLATE/Bug report form.yml

```diff
@@ -0,0 +1,127 @@
+name: Report a problem/bug
+title: '[Problem/Bug]: '
+description: File a report for something not working.
+labels: ["bug"]
+body:
+- type: markdown
+  attributes:
+    value: 
+      Thanks for taking the time to fill out this report. Here is [an example](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3648) of a good bug report that will help us in accurately identifying the issue.
+- id: what-happened
+  type: textarea
+  attributes:
+    description: Please tell us what's not working as expected. Please include callstacks and error messages if possible.
+    label: What happened?
+    placeholder: Please describe the issue you encountered. Callstacks and error messages are helpful.
+  validations:
+    required: true
+- id: importance
+  type: dropdown
+  attributes:
+     label: Importance
+     description: May you help us understand the level of importance of this ask to you?
+     multiple: false
+     options:
+       - ----Please select----
```

Comment:
Does this mean Please Select is one of the options you can choose? Should we remove this? (Also applies to one other place in this doc)

Reply:
Yes. Otherwise it will look like this and my concern is people might just use the default one without selecting. 
<img width="176" alt="image" src="https://github.com/MicrosoftEdge/WebView2Feedback/assets/6320831/38f48f98-0384-430a-b2b7-324bb8c7a435">